common/buildcraft/core/BCCoreItems.java

```diff
@@ -34,15 +34,19 @@ public static void preInit() {
         gearIron = ItemManager.register(new ItemBC_Neptune("item.gear.iron"));
         gearGold = ItemManager.register(new ItemBC_Neptune("item.gear.gold"));
         gearDiamond = ItemManager.register(new ItemBC_Neptune("item.gear.diamond"));
-        if (DEV) diamondShard = ItemManager.register(new ItemBC_Neptune("item.diamond.shard"));
+        if (BCLib.DEV) {
+            diamondShard = ItemManager.register(new ItemBC_Neptune("item.diamond.shard"));
+        }
         paintbrush = ItemManager.register(new ItemPaintbrush_BC8("item.paintbrush"));
         list = ItemManager.register(new ItemList_BC8("item.list"));
-        if (DEV) mapLocation = ItemManager.register(new ItemMapLocation("item.map_location"));
-        if (DEV) markerConnector = ItemManager.register(new ItemMarkerConnector("item.marker_connector"));
-        if (DEV) goggles = ItemManager.register(new ItemGoggles("item.goggles"));
-
-        if (DEV) ItemManager.register(new ItemVolumeCuboid("item.volume_marker"));
-
+        if (BCLib.DEV) {
+            mapLocation = ItemManager.register(new ItemMapLocation("item.map_location"));
+            markerConnector = ItemManager.register(new ItemMarkerConnector("item.marker_connector"));
+        }
+        volumeBox = ItemManager.register(new ItemVolumeBox("item.volume_box"));
```

Comment:
Volume is needed for filling planner, filler, architect table, and this means that it's needed for builder.